production/ci/integration/run.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+
+set -e -o pipefail
+
+if [ ! -e ROOT ] ; then
```

Comment:
detect `.git` if you must instead of adding empty file to confuse users

Reply:
Then this won't work if we strip off git metadata somewhere in the pipeline (probable at some point).

I can rename it to .ROOT if you prefer.

Reply:
It's a failsafe for the users anyways, it's detecting user error when using our packages right? I think .git is enough for most cases. empty ROOT file is confusing/hacky and I have never seen it used anywhere else